ebos/eclproblem.hh

```diff
@@ -695,15 +699,18 @@ public:
             checkDeckCompatibility_();
 
         // write the static output files (EGRID, INIT, SMSPEC, etc.)
-        if (enableEclOutput_)
+        if (enableEclOutput_ && !initconfig.restartRequested())
```

Comment:
I didn't pay attention to it when the PR was first submitted, but this particular change (adding `!restartRequested()` here) has the side effect of not creating the `.EGRID` or `.INIT` files for a restarted case.  That, in turn, means that it will be impossible to visualise 3D vectors (e.g., pressure) in postprocessors like ResInsight when running restarted simulations.

Is that intentional?  If not, why did you have to add this conditional?

Reply:
It was intentional. I thought you didn't want to overwrite the .INIT files with the restarted values. You know best what't the expected behavior should be like. Do you want me to make a PR that revert this change or would you do it?

Reply:
> I thought you didn't want to overwrite the .INIT files with the restarted values.

Restarted runs have case names that differ from those of the base runs.  E.g, a base run might be called `CASE.DATA` and then the restart run might be called `CASE-RST.DATA`.  In that case we need a `CASE-RST.INIT` and a `CASE-RST.EGRID` in order to visualise the 3D vectors in ResInsight.

> Do you want me to make a PR that revert this change or would you do it?

It depends on your workload and timing.  If you're able to do so soon, then I'd appreciated it if you could make the change.  Otherwise I'll do it towards the end of the day today (Monday 3rd of June).

Reply:
I can make the PR asap.

Reply:
Done. i.e #1873. I have verified that it outputs a INIT and GRID file and that I am able to load it in ResInsight.